src/app/components/expeditions/FimsExpeditionController.js

```diff
@@ -8,7 +8,9 @@ export class FimsExpeditionController {
   }
 
   exportData(projectId, expedition) {
-    this.DataService.exportData(projectId, expedition.expeditionCode);
+    this.loading = true;
+    this.DataService.exportData(projectId, expedition.expeditionCode)
+    .then(() => {this.loading = false});
```

Comment:
this should be `.finally` b/c if the export fails, then the loading sign will still show